pom.xml

```diff
@@ -25,6 +30,11 @@
 			<artifactId>utils</artifactId>
 			<version>1.07.00</version>
 		</dependency>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-digester3</artifactId>
```

Comment:
_nit_ not following the original spacing in the file.

Reply:
the file was already mixing tabs and blanks, as you can see in the parent part, but I can do that